### PR TITLE
Daily update sources + Fix

### PR DESCRIPTION
**Pull request by dragonx943**
- Change BlogTruyenVN domain (updated and still keep broken tags)
- Still keep and add Broken tag for BlogTruyen parser (closed site, should be added https://github.com/KotatsuApp/kotatsu-parsers/issues/39)
- Try to fix TopTruyenViet, DocTruyen3Q (https://github.com/KotatsuApp/kotatsu-parsers/issues/1115), LxManga (https://github.com/KotatsuApp/kotatsu-parsers/issues/1138)
- All Vietnamese sources domain have been updated

### CODE REVIEW

**Comment by dragonx943** [comment]
> * Remove BlogTruyenVN parser (this source has been sold and changed to: [BumCheo #1135](https://github.com/KotatsuApp/kotatsu-parsers/issues/1135)), should be added: [List of sources that will no longer work #39](https://github.com/KotatsuApp/kotatsu-parsers/issues/39)
> * Fix TopTruyenViet, DocTruyen3Q and CuuTruyen (Tags again)
> * All Vietnamese sources domain have been checked

@Koitharu

File: src/main/kotlin/org/koitharu/kotatsu/parsers/site/wpcomics/vi/DocTruyen3Q.kt [hunk 1]
```diff
@@ -1,12 +1,123 @@
...
+                append(page.toString())
+            }
+        }
+        val doc = webClient.httpGet(url).parseHtml()
+        return parseMangaList(doc)
+    }
+
+    private fun parseMangaList(doc: Document): List<Manga> {
+        return doc.select("div.list-story-item").map { div ->
+            val href = div.selectFirst("h3.title-book a")?.attrAsRelativeUrl("href") ?: div.parseFailed("Manga link not found")
```
Comment (Koitharu) [inline src/main/kotlin/org/koitharu/kotatsu/parsers/site/wpcomics/vi/DocTruyen3Q.kt:49]:
```suggestion
            val href = div.selectFirstOrThrow("h3.title-book a").attrAsRelativeUrl("href")
```

File: src/main/kotlin/org/koitharu/kotatsu/parsers/site/wpcomics/vi/DocTruyen3Q.kt [hunk 2]
```diff
@@ -1,12 +1,123 @@
...
+        }
+    }
+
+    override suspend fun getDetails(manga: Manga): Manga {
+        val doc = webClient.httpGet(manga.url.toAbsoluteUrl(domain)).parseHtml()
+        return manga.copy(
+            tags = doc.select("li.kind a").mapNotNullToSet { a ->
+                MangaTag(
+                    key = a.attr("href").substringAfterLast('/'),
+                    title = a.text().trim(),
```
Comment (Koitharu) [inline src/main/kotlin/org/koitharu/kotatsu/parsers/site/wpcomics/vi/DocTruyen3Q.kt:73]:
```suggestion
                    title = a.text().toTitleCase(sourceLocale),
```

File: src/main/kotlin/org/koitharu/kotatsu/parsers/site/wpcomics/vi/DocTruyen3Q.kt [hunk 3]
```diff
@@ -1,12 +1,123 @@
...
+                source = source,
+                isNsfw = isNsfwSource,
+            )
+        }
+    }
+
+    override suspend fun getDetails(manga: Manga): Manga {
+        val doc = webClient.httpGet(manga.url.toAbsoluteUrl(domain)).parseHtml()
+        return manga.copy(
+            tags = doc.select("li.kind a").mapNotNullToSet { a ->
```
Comment (Koitharu) [inline src/main/kotlin/org/koitharu/kotatsu/parsers/site/wpcomics/vi/DocTruyen3Q.kt:70]:
```suggestion
            tags = doc.select("li.kind a").mapToSet { a ->
```

File: src/main/kotlin/org/koitharu/kotatsu/parsers/site/wpcomics/vi/TopTruyenViet.kt [hunk 4]
```diff
@@ -1,12 +1,69 @@
...
+    override val selectPage = "div.page-chapter > img, div.chapter-content > img"
+    override val selectChapter = "div.list-chapter > a"
+    override val selectDate = "div.col-xs-4"
+    override val datePattern = "dd/MM/yyyy"
+
+    override suspend fun getChapters(doc: Document): List<MangaChapter> {
+        return doc.select(selectChapter).mapChapters(reversed = true) { i, a ->
+            val href = a.attrAsRelativeUrl("href")
+            val dateText = a.selectFirst(selectDate)?.text()
+            val dateFormat = SimpleDateFormat(datePattern, sourceLocale)
```
Comment (Koitharu) [inline src/main/kotlin/org/koitharu/kotatsu/parsers/site/wpcomics/vi/TopTruyenViet.kt:38]: Move out of the loop

**Review by Koitharu — APPROVED** [review]

**Pull request merged**